platform/security/idp/security-idp-client/src/main/java/org/codice/ddf/security/idp/client/IdpMetadata.java

```diff
@@ -61,8 +62,9 @@
     public void setMetadata(String metadata)
             throws WSSecurityException, XMLStreamException, SAMLException, IOException {
         MetadataConfigurationParser metadataConfigurationParser = new MetadataConfigurationParser(
-                Collections.singletonList(metadata));
-        entryDescriptions = metadataConfigurationParser.getEntryDescriptions();
+                Collections.singletonList(metadata),
+                ed -> entryDescriptions.put(ed.getEntityID(), ed));
```

Comment:
Not that I can see. On the server side, we're building a new `EntityInformation` with the `EntityDescriptor` and the `build` method on that can return null. The ED shouldn't ever be null here.